src/Rx.DOM.ts

```diff
@@ -1,167 +1,22 @@
-/* tslint:disable:no-unused-variable */
-// Subject imported before Observable to bypass circular dependency issue since
-// Subject extends Observable and Observable references Subject in it's
-// definition
-import {Subject} from './Subject';
-/* tslint:enable:no-unused-variable */
-import {Observable} from './Observable';
-
-// statics
-/* tslint:disable:no-use-before-declare */
-import './add/observable/combineLatest';
-import './add/observable/concat';
-import './add/observable/merge';
-import './add/observable/bindCallback';
-import './add/observable/defer';
-import './add/observable/empty';
-import './add/observable/forkJoin';
-import './add/observable/from';
-import './add/observable/fromArray';
-import './add/observable/fromEvent';
-import './add/observable/fromEventPattern';
-import './add/observable/fromPromise';
-import './add/observable/interval';
-import './add/observable/never';
-import './add/observable/range';
-import './add/observable/throw';
-import './add/observable/timer';
-import './add/observable/zip';
+export * from './Rx';
```

Comment:
@masaeedu you saying we need to import and export it?  If I just change it to import things break.  If I only export, I get correct typings my editor, and everything seems to be working fine.